src/settings.yml.sample

```diff
@@ -117,3 +117,76 @@ pinanas:
 
       # Send emails as. Type: string. Example: "contact@{{ pinanas.domain }}".
       sender:
+
+
+  applications:
```

Comment:
Include the default list of applications here:
- It must contains all application per default
- It should be preceded by a comment: the field definition:
```yaml
  applications:
    # List of enabled applications. Type: list of strings. Default: none (all enabled). Example:
    # - dhcpd
    # - traefik
    - <here: list of all applications>
```